sktime/forecasting/base/adapters/_neuralforecast.py

```diff
@@ -332,15 +358,47 @@ def _fit(
 
         maximum_forecast_horizon = fh.to_relative(self.cutoff)[-1]
         self._forecaster = self._instantiate_model(maximum_forecast_horizon)
-
         self._forecaster.fit(df=train_dataset, verbose=self.verbose_fit)
 
         return self
 
+    def _get_id_idx(self, indices):
```

Comment:
Can you please add docstring (and pereferably type hints) to explain what does this do?

Reply:
Docstring and hype hints are added.